localstack/services/apigateway/apigateway_listener.py

```diff
@@ -135,7 +133,7 @@ def forward_request(self, method, path, data, headers):
                 else:
                     msg = 'API Gateway action uri "%s" not yet implemented' % uri
                     LOGGER.warning(msg)
-                    return make_error(msg, 404)
+                    return make_error(msg, 407)
```

Comment:
HTTP code `407` is "Proxy Authentication Required" - wondering what is the rationale behind this change?

Reply:
This was for debugging so I could see which make_error() was being called by changing the codes.  It's a mistake; I'll revert it.